### ISSUE TITLE
zookeeper pod can't start

### ISSUE THREAD

**Issue by stalek71** [SigNoz/charts#90]
I have a node group based on arm64 and all pods except zookeper are properly constructed.
How to make it working? Is there any elem in values.yaml I could override? 
It should choose arm64 image out of the box, but it looks it runs x64/x86 image...

<img width="386" alt="image" src="https://user-images.githubusercontent.com/6651025/195089617-66f979d3-99d3-47e3-8633-b322aed8e7d3.png">

**Comment by pranay01** [comment]
@stalek71 Can you share which version of SigNoz are you using?

**Comment by stalek71** [comment]
I just ran it through the latest chart version so I 'm not sure what the
version was...

wt., 11 paź 2022, 19:56 użytkownik Pranay Prateek ***@***.***>
napisał:

**Comment by prashant-shahi** [comment]
@stalek71 We don't entirely support ARM. There are few components which causes issues like alertmanager and zookeeper.
We will be working on supporting it near future.

_Note: Apple chips (M1/M2) works fine for Docker Standalone since they internally handle support for those component binaries._

**Comment by stalek71** [comment]
I was trying to run it on azure. I was able to start zookeeper 3.8 on arm
without any problems (using bitnami helm), but disabling the construction
of zookeeper to use external one in your helm doesn't work as well (you
always construct it)

śr., 12 paź 2022, 06:25 użytkownik Prashant Shahi ***@***.***>
napisał:

**Comment by prashant-shahi** [comment]
@stalek71 Yeah, that could work. Any contributions would be very much appriciated.

Also, something to note here is that zookeeper is only mandatory in case of ClickHouse cluster with multiple replicas/shards.
It is not a "must have" for default signoz installation where only single instance of clickhouse is present.
